service/src/main/java/com/epam/esm/services/constants/CertificateSortingParameters.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.esm.services.constants;
+
+import com.epam.esm.persistence.constants.CertificateColumns;
+import com.epam.esm.persistence.constants.TagColumns;
+
+public enum CertificateSortingParameters {
+    SORT_BY_ID(CertificateColumns.ID.getValue()),
+    SORT_BY_NAME(CertificateColumns.NAME.getValue()),
+    SORT_BY_DESCRIPTION(CertificateColumns.DESCRIPTION.getValue());
+
+    String value;
+
+    CertificateSortingParameters(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public static CertificateSortingParameters getEntryByParameter(String s) {
```

Comment:
String s - not crear of the sense of parameter "s". What is this? naming issue.

Reply:
renamed to parameter

Reply:
renamed to "parameter"